dvc/command/diff.py

```diff
@@ -167,6 +169,15 @@ def add_parser(subparsers, parent_parser):
         help=DIFF_DESCRIPTION,
         formatter_class=argparse.RawDescriptionHelpFormatter,
     )
+    diff_parser.add_argument(
+        "-t",
+        "--target",
```

Comment:
How about `--targets`, since we have it in `dvc plots/metrics` already?

Reply:
@efiop makes sense, incorporated in fa6c3b4d3868b240c12200c8dc8082e1bfc4ab3b.